### ISSUE TITLE
mask-rcnn seg  AP is low

### ISSUE THREAD

**Issue by AubeZ** [open-mmlab/mmdetection#7916]
sorry,my english is poor.
i don't sure my question is belong to this block.
i use .configs/mask_rcnn/mask_rcnn_r101_caffe_fpn_mstrain-poly_3x_coco.py, and my training dataset is pascal voc (i translate pascal voc into coco and use it to detection and segmentation) 
lr_config = dict(step=[20, 30,40])
runner = dict(type='EpochBasedRunner', max_epochs=50)
device ='cuda'
samples_per_gpu=2,
optimizer = dict(type='SGD', lr = 0.005, momentum=0.9, weight_decay=0.0001)

i have 232660+ images, 
MyDataset Train dataset with number of images 2326, and instance counts: 
+------------------+-------+-------------+-------+------------+-------+----------------+-------+----------------+-------+
| category         | count | category    | count | category   | count | category       | count | category       | count |
+------------------+-------+-------------+-------+------------+-------+----------------+-------+----------------+-------+
| 0 [aeroplane]    | 162   | 1 [bicycle] | 147   | 2 [bird]   | 234   | 3 [boat]       | 183   | 4 [bottle]     | 236   |
| 5 [bus]          | 193   | 6 [car]     | 358   | 7 [cat]    | 223   | 8 [chair]      | 448   | 9 [cow]        | 231   |
| 10 [diningtable] | 137   | 11 [dog]    | 221   | 12 [horse] | 163   | 13 [motorbike] | 162   | 14 [person]    | 1375  |
| 15 [pottedplant] | 280   | 16 [sheep]  | 276   | 17 [sofa]  | 178   | 18 [train]     | 143   | 19 [tvmonitor] | 151   |
+------------------+-------+-------------+-------+------------+-------+----------------+-------+----------------+-------+

my bbox AP is here:
bbox_mAP: 0.2800, bbox_mAP_50: 0.5780, bbox_mAP_75: 0.2210, bbox_mAP_s: 0.1120, bbox_mAP_m: 0.2490, bbox_mAP_l: 0.3310, bbox_mAP_copypaste: 0.280 0.578 0.221 0.112 0.249 0.331, segm_mAP: 0.0130, segm_mAP_50: 0.0590, segm_mAP_75: 0.0010, segm_mAP_s: 0.0010, segm_mAP_m: 0.0050, segm_mAP_l: 0.0180, segm_mAP_copypaste: 0.013 0.059 0.001 0.001 0.005 0.018

why my segm AP is so low ?
can you help me?
if anyone want to connect me ,you can add me wechat :18220575565

**Comment by BIGWangYuDong** [comment]
Did you mean you use your own dataset, or use PASCAL VOC dataset?
If you use your own dataset, it is suggested to get your dataset baseline at faster rcnn-r50 or mask rcnn-r50

Also please note, pstrain-poly use repeatdataset(times=3). So if you set max_epoch=50 that means you trained 150 epoches

**Comment by AubeZ** [comment]
thanks for you help.
i use my own dataset ,i createit by copy COCODataset，and change the classes to pascal voc dataset class
now i chang my baseline as mask rcnn-r101
and i change some parameter (use pascal0712.py)

_base_ = ['../configs/_base_/models/mask_rcnn_r50_fpn.py',
          '../configs/_base_/default_runtime.py'
]

img_norm_cfg = dict(
    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
train_pipeline = [
    dict(type='LoadImageFromFile'),
    dict(type='LoadAnnotations', with_bbox=True, with_mask=True),
    dict(type='Resize', img_scale=(1000, 600), keep_ratio=True),
    dict(type='RandomFlip', flip_ratio=0.5),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='Pad', size_divisor=32),
    dict(type='DefaultFormatBundle'),
    dict(type='Collect', keys=['img', 'gt_bboxes', 'gt_labels', 'gt_masks']),
]
test_pipeline = [
    dict(type='LoadImageFromFile'),
    dict(
        type='MultiScaleFlipAug',
        img_scale=(1000, 600),
        flip=False,
        transforms=[
            dict(type='Resize', keep_ratio=True),
            dict(type='RandomFlip'),
            dict(type='Normalize', **img_norm_cfg),
            dict(type='Pad', size_divisor=32),
            dict(type='ImageToTensor', keys=['img']),
            dict(type='Collect', keys=['img']),
        ])
]
dataset_type = 'MyDataset'
data_root = ''
data = dict(
    samples_per_gpu=2,
    workers_per_gpu=0,
    train=dict(
        type=dataset_type,
        img_prefix='./data/coco/train2014/',
        # classes=classes,
        ann_file='./data/coco/annotations/instances_train2014.json',
        pipeline=train_pipeline),
    val=dict(
        type=dataset_type,
        img_prefix='./data/coco/val2014/',
        # classes=classes,
        ann_file='./data/coco/annotations/instances_val2014.json',
        pipeline=test_pipeline),
    test=dict(
        type=dataset_type,
        img_prefix='./data/coco/val2014/',
        # classes=classes,
        ann_file='./data/coco/annotations/instances_val2014.json',
        pipeline=test_pipeline))
evaluation = dict(metric=['bbox', 'segm'])


device ='cuda'
# # 我们需要对头中的类别数量进行修改来匹配数据集的标注
model = dict(
    backbone=dict(
        depth=101,
        init_cfg=dict(type='Pretrained',
                      checkpoint='torchvision://resnet101')),
        roi_head=dict(
        bbox_head=dict(num_classes=20),
        mask_head=dict(num_classes=20)))

# optimizer
optimizer = dict(type='SGD', lr=0.0025, momentum=0.9, weight_decay=0.0001)
optimizer_config = dict(grad_clip=None)
# learning policy
# actual epoch = 3 * 3 = 9
lr_config = dict(policy='step', step=[3])
# runtime settings
runner = dict( type='EpochBasedRunner', max_epochs=120)  # actual epoch = 4 * 3 = 12
